src/Service/PopulatingManager.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\Housing;
+
+class PopulatingManager
+{
+
+    const FIXED_COLUMNS = [
+        'nom-de-la-copropriete',
+        'cp',
+        'nombre-de-lots',
+        'honoraires',
+    ];
+
+    /**
+     * @var ParsingManager
+     */
+    private $parsing;
+
+    public function __construct(ParsingManager $parsing)
+    {
+        $this->parsing = $parsing;
+    }
+
+    private function stringToInteger(array $activities)
+    {
+        return $activities = array_map('intval', $activities);
+    }
+
+    public function populateHousing(array $housings)
+    {
+        $userHousings = [];
+        foreach ($housings as $property) {
+            array_shift($property);
+            $property = $this->parsing->slugArrayKey($property);
+
+
+            $property['immeuble-de-moins-de-2-ans'] =
+                $this->parsing->convertToBoolean($property['immeuble-de-moins-de-2-ans']);
+            $activities = $this->stringToInteger(array_slice($property, count(self::FIXED_COLUMNS), null, true));
+
+            $property[self::FIXED_COLUMNS[3]] = str_replace(',', '.', $property[self::FIXED_COLUMNS[3]]);
+
+            //this is a whitespace used for numbers (different of the usual whitespace)
+            //careful when modifying this line
+            //unicode(\u202F)
+            $property[self::FIXED_COLUMNS[3]] = ltrim(str_replace(' ', '', $property[self::FIXED_COLUMNS[3]]));
```

Comment:
ça serait plus pertinent de faire une méthode par ex moneyToFloat (pourquoi pas dans votre ParsingService qui va etre un service qui va faire tous les petits ajustements sur vos valeurs récupécrée)
et vous auriez un truc comme $this->parsing->moneyToFloat($property[self::FIXED_COLUMNS['HONORARY'])